packages/lodestar/src/api/impl/validator/validator.ts

```diff
@@ -75,6 +75,7 @@ export class ValidatorApi implements IValidatorApi {
     graffiti = ""
   ): Promise<BeaconBlock> {
     const validatorIndex = (await this.chain.getHeadEpochContext()).pubkey2index.get(validatorPubkey);
+    if (validatorIndex === undefined) throw Error("Validator pubKey not in epochCtx");
```

Comment:
Not in this PR but we should make this error more sensible so we can trow 400 in case of this error.

Reply:
Opened https://github.com/ChainSafe/lodestar/issues/1293